File: II sem/Programowanie Obiektowe/Projekty/projekt2/po3/src/Owca.java
public class Owca extends Zwierze {
    public Owca(String ikonka, int x_pos, int y_pos, int sila, int inicjatywa, Swiat swiat) {
        super(ikonka, x_pos, y_pos, sila, inicjatywa, swiat);
    }

    public Owca(Organizm organizm) {
        super(organizm);
    }

    @Override
    Organizm kopiuj(Organizm organizm) {
        return new Owca(organizm);
    }


}

File: II sem/Programowanie Obiektowe/Projekty/projekt1/POproject1/Owca.cpp
#include "Owca.h"

Owca::Owca(int x, int y) : Zwierze(EMOJI ? u8"O" : "Ow", 4, 4, x, y)
{

}

Owca* Owca::potomek()
{
	return new Owca(*this);
}